window.go

```diff
@@ -116,9 +117,19 @@ type WindowOptions struct {
 	Y                      *int            `json:"y,omitempty"`
 
 	// Additional options
-	Custom *WindowCustomOptions `json:"custom,omitempty"`
-	Load   *WindowLoadOptions   `json:"load,omitempty"`
-	Proxy  *WindowProxyOptions  `json:"proxy,omitempty"`
+	Custom     *WindowCustomOptions `json:"custom,omitempty"`
+	Load       *WindowLoadOptions   `json:"load,omitempty"`
+	Proxy      *WindowProxyOptions  `json:"proxy,omitempty"`
+	AppDetails *WindowAppDetails    `json:"appDetails,omitempty"`
+}
+
+// WindowAppDetails represents window app details
+type WindowAppDetails struct {
```

Comment:
Could you add this link `https://github.com/electron/electron/blob/v4.0.1/docs/api/browser-window.md#winsetappdetailsoptions-windows` as an additional commentary below `// WindowAppDetails represents window app details`?